osu.Game/Utils/StatelessRNG.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Utils
+{
+    /// Provides a fast stateless function that can be used in randomly-looking visual elements.
```

Comment:
These general descriptions should be in `<summary></summary>` tags (goes for both the class and the public methods).

Also, would rename public members to match `RNG`:

* `Get()` -> `Next()`
* `GetSingle()` -> `NextSingle()`